patches/minecraft/net/minecraft/client/renderer/RenderGlobal.java.patch

```diff
@@ -124,7 +124,30 @@
          if (this.field_72777_q.field_71441_e.field_73011_w.func_76569_d())
          {
              if (this.field_72777_q.field_71474_y.func_181147_e() == 2)
-@@ -1852,7 +1875,8 @@
+@@ -1702,9 +1725,11 @@
+             {
+                 RenderChunk renderchunk1 = iterator.next();
+                 boolean flag1;
++                long k1 = p_174967_1_ - System.nanoTime();
+ 
+                 if (renderchunk1.func_188281_o())
+                 {
++                    if (k1 < 0L) continue;
```

Comment:
config

Reply:
This is just here because the `break` below is now conditional, and I don't want to change the semantics of this block, only the later one

Reply:
The original version, for context:
```java
while (iterator.hasNext())
{
    RenderChunk renderchunk1 = iterator.next();
    boolean flag1;

    if (renderchunk1.needsImmediateUpdate())
    {
        flag1 = this.renderDispatcher.updateChunkNow(renderchunk1);
    }
    else
    {
        flag1 = this.renderDispatcher.updateChunkLater(renderchunk1);
    }

    if (!flag1)
    {
        break;
    }

    renderchunk1.clearNeedsUpdate();
    iterator.remove();
    long k1 = finishTimeNano - System.nanoTime();

    if (k1 < 0L)
    {
        break;
    }
}
```